### DISCUSSION TITLE
Настройка внешних форм

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#490] [Как это сделать?] [answered]
Добрый день, изучив документацию по внешним формам, не совсем понял следующее: 

1. ![1](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/cfe03022-e8ba-45df-8e8b-b2bb761e0a50)
**Как взаимодествовать с css ?**,  **За что отвечает Section statuses code ?**

В документации есть следующее: 
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/53f6b785-97fa-4cb4-8dc1-e0743faf7a24)
Не совсем понятно, как именно к определенному полю "name" допустим "Type1"  задать тот или иной параметр в css.

2. Возможно ли, взаимодействии быстрой формы с друг другом? Т.е. в 1 вкладке ввел данные, нажимешь на кнопку и открывается 2 вкладка и т.п.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. По CSS детально не скажу, но вот пример: https://calc-totum.totum.online/Forms/ccalc

```css
.MuiCheckbox-colorSecondary.Mui-checked {
  color: #33aa66
}
```

2. Не знаю зачем это именно для _быстрой_ формы — тк _быстрая_ это строка добавления в таблицу. Обычно это делается для просто форм, которые на основе временной таблицы. Смотрите функции LinkToForm и linkToQuickForm. Они позволяют получить url с шифрованными параметрами. Далее вы отправляете пользователя по linkToScript по сгенерированному url.

Вот пример: https://control.ttmapp.ru/Forms/order-plus по кнопке Заказать происходит переход во вторую форму с передачей параметров.

**Reply by ttm-alexeypolunin** [reply]
Section statuses code — продвинутый функционал управления вкладками типа как в примере с Model 3. Документации по нему нет и будет ток когда заработают коммерческие лицензии и кто-то за них будет платить. А пока мы работаем ассенизаторами и иногда пишем тотум — доков по этой истории не будет. Сорри :)

Напоминаю, что формы будут выпилены из MIT-версии при переходе на 5 версию.
